vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -195,6 +195,7 @@ type internal FSharpCheckerWorkspaceServiceFactory
 
 type
     [<Guid(FSharpCommonConstants.packageGuidString)>]
+    [<ProvideLanguageEditorOptionPage(typeof<Options.IntelliSenseOptionPage>, "F#", null, "IntelliSense", "6008")>]
```

Comment:
How do you know/Where did you find the numbers(keys?) we're supposed to use for these? I'd like to get started on making a few of them too